### ISSUE TITLE
Not working on any non-bash shell

### ISSUE THREAD

**Issue by mwalzer** [PaulBrack/Yamato#109]
```
Yamato.Console: line 1: can't create ??@@: Read-only file system
Yamato.Console: line 1: ELF: not found
Yamato.Console: line 2: syntax error: unexpected word (expecting ")")
```

**Comment by mwalzer** [comment]
tested alpine's `ash` so far only

**Comment by PaulBrack** [comment]
This seems like an environment issue - not sure I know how to resolve this

**Comment by mwalzer** [comment]
@Ozzard any experience with non-bash .dotnet? Would be a shame if the standalone would be limited* to some distros only. Worse if root cause is in the env, because that would mean that in other bash based OS it could not work as well. I can't help more as I know too little about the build process of yamato.
(*: limited as in not without serious efforts to install )

**Comment by Ozzard** [comment]
@mwalzer Could you send me your Dockerfile (or point to it on here if it's committed) and I'll take a look? Could you also send me the command line that you're giving to ash? Alpine is a supported platform for .Net Core 2.1+ (Yamato is on 2.2) and there are base containers with the .Net Core prereqs installed available from Microsoft, so I suspect all of our experimentation so far hasn't quite worked!

**Comment by mwalzer** [comment]
Sure @Ozzard here you go:
[yamato.Dockerfile.txt](https://github.com/PaulBrack/Yamato/files/4149094/yamato.Dockerfile.txt)
[yamato.sdef.txt](https://github.com/PaulBrack/Yamato/files/4149095/yamato.sdef.txt)
[yamato_alpine.Dockerfile.txt](https://github.com/PaulBrack/Yamato/files/4149096/yamato_alpine.Dockerfile.txt)
[yamato_alpine.sdef.txt](https://github.com/PaulBrack/Yamato/files/4149097/yamato_alpine.sdef.txt)

I added both the Ubuntu version as well as the Alpine version and both also as singularity container versions in case that might be of any interest.

**Comment by Ozzard** [comment]
I believe this is addressed by #119. This uses Microsoft's dependency base image (based on Debian) and builds it up.  Note that the Alpine image could not work with the standard Linux build - see https://github.com/dotnet/core/issues/1689 for the gory details.  We should work on getting a non-standard standalone build made for Alpine, as it's a much smaller image.

Built versions of v1.0.0-b available at https://hub.docker.com/r/ozzard/yamato - please test

**Comment by mwalzer** [comment]
the container seems to have a blank entry point (Yamato.Console w/o parameters). I had to override it with `--entrypoint=/bin/bash` otherwise I wouldn't know how to convey my input parameter values
